src/org/usfirst/frc/team4915/stronghold/vision/imgExplore.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+#
+#
+# imgExplore.py
+#   a python/opencv utility for perusing video or image streams.
+#   Performance issues can be resolved by migrating algorithms to
+#   gpu. (VisionWorks, Cuda, etc are available optimization avenues)
+#
+#   db: 1/28/16, for spartronics 4915
+#
+#
+
+from __future__ import print_function
+import cv2
+import os.path
+import sys, getopt
+import common
+import numpy as np
+
+from multiprocessing.pool import ThreadPool
+from collections import deque
+
+fnpat = '/home/knx32542/Downloads/RealFullField/%d.jpg'
```

Comment:
this shouldn't be hardcoded -- see https://stackoverflow.com/questions/4028904/how-to-get-the-home-directory-in-python